### ISSUE TITLE
Create the Product Security Committee 

### ISSUE THREAD

**Issue by philips** [kubernetes/steering#89]
tl;dr @kubernetes/steering-committee LGTM this issue to approve the [Kubernetes Product Security Team](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#product-security-team-pst) becoming the Product Security Committee.

The product security team was formed to handle disclosures to security@kubernetes.io and document disclosure processes at kubernetes.io/security.

For the last 1.5 years it has been operating as a nebulous team as part of SIG Release which was the recommendation of the steering committee at the time. Today though, we have a [committee creation process](https://github.com/kubernetes/community/blob/master/governance.md#committees).

LGTM this issue to form the Kubernetes Product Security Committee.

**Comment by dims** [comment]
LGTM 👍

**Comment by spiffxp** [comment]
LGTM

**Comment by tpepper** [comment]
LGTM from SIG Release side

**Comment by justaugustus** [comment]
Another LGTM from SIG Release.
@spiffxp -- thanks for getting us closure on this so quickly! :)

**Comment by dims** [comment]
LGTM

**Comment by dims** [comment]
Do we need to update some metadata somewhere?

**Comment by philips** [comment]
@parispittman Do you know where committees live? Or maybe @michelleN who has been helping get the CoCC one going?

**Comment by michelleN** [comment]
@philips The CoCC info lives in the kubernetes/community repo [here](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct)

**Comment by philips** [comment]
@spiffxp as part of the GitHub management team can you create the security committee github.com/kubernetes/security similar to github.com/kubernetes/steering? We have a number of documents that we need to manage like subscription lists for CVE disclosure which currently live here: https://github.com/kubernetes/sig-release/tree/master/security-release-process-documentation

**Comment by cblecker** [comment]
https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-repo&template=repo-create.md

**Comment by philips** [comment]
Thanks @cblecker.

Repo request here: https://github.com/kubernetes/org/issues/520

**Comment by joelsmith** [comment]
Now that the repo has been created, I've opened a PR to copy the docs from sig-release:
https://github.com/kubernetes/security/pull/1
And another one to document the new committee in k/community:
https://github.com/kubernetes/community/pull/3311

**Comment by nikhita** [comment]
Looks like this is done? :)

- https://github.com/kubernetes/security exists with docs for security-related processes
- The committee exists formally in the community repo: https://github.com/kubernetes/community/tree/master/committee-product-security

**Comment by philips** [comment]
Yes, the last thing is to clean up the sig release repo and put in redirects from the old docs with deletion times.

If anyone has time to get to it before me I would appreciate it :)

**Comment by nikhita** [comment]
> If anyone has time to get to it before me I would appreciate it :)

Created https://github.com/kubernetes/sig-release/pull/537. Let's wrap this up. :)

**Comment by nikhita** [comment]
https://github.com/kubernetes/sig-release/pull/537 merged :tada: 

Closing.
/close

**Comment by philips** [comment]
thanks @nikhita !!!
